src/framework/ui/view/uitheme.cpp

```diff
@@ -32,9 +32,11 @@
 #include <QToolBar>
 #include <QTextEdit>
 #include <QVariant>
+#include <math.h>
 
 #include "log.h"
 
+
```

Comment:
this extra line cause the code style check to fail

Reply:
Yes, fixed already :)

Reply:
You were quicxker in fixing that I in reporting ;-)